Le module principal de Pyodide-Qt : le Pyodide amont plus embind, dont le module latéral de Qt (pyside_agrege.so,
construire.sh, phase dynamique) appelle les fonctions JavaScript, que seul le module principal peut fournir.

diff --git a/Makefile.envs b/Makefile.envs
index d0b5c79..050335d 100644
--- a/Makefile.envs
+++ b/Makefile.envs
@@ -197,7 +197,8 @@ export MAIN_MODULE_LDFLAGS= $(LDFLAGS_BASE) \
 	-lhtml5_webgl.js \
 	-lsdl.js \
 	-sMAX_WEBGL_VERSION=2  \
-	-sGL_WORKAROUND_SAFARI_GETCONTEXT_BUG=0
+	-sGL_WORKAROUND_SAFARI_GETCONTEXT_BUG=0 \
+	-lembind
 
 EXPORTS=_main \
    ,_free \
